torch/_C/_onnx.pyi

```diff
@@ -23,6 +23,8 @@ class TensorProtoDataType(Enum):
     COMPLEX64 = ...
     COMPLEX128 = ...
     BFLOAT16 = ...
+    FP8E4M3 = ...
+    FP8E5M2 = ...
```

Comment:
Could you take this chance and also add the other [2 missing fp8 types](https://github.com/onnx/onnx/blob/main/onnx/onnx.proto3#L512-L521)?

Should be little extra work and a full fp8 support would be available :)

```
    # From ONNX spec
    FLOAT8E4M3FN = 17;    // float 8, mostly used for coefficients, supports nan, not inf 
    FLOAT8E4M3FNUZ = 18;  // float 8, mostly used for coefficients, supports nan, not inf, no negative zero 
    FLOAT8E5M2 = 19;      // follows IEEE 754, supports nan, inf, mostly used for gradients
    FLOAT8E5M2FNUZ = 20;  // follows IEEE 754, supports nan, inf, mostly used for gradients, no negative zero
```

Reply:
My plan was to add all flavors, but this [MR](https://github.com/pytorch/pytorch/commit/b64bd4a5dd9ee21d269997c4a28f6389870cd540) added only fp8e4m3fn and fp8e5m2 to torch, so currently we can support only this datatypes export